installer/bundle/bundle_skel.sh

```diff
@@ -387,6 +411,58 @@ install_if_program_does_not_exist_on_system()
     fi
 }
 
+install_troubleshooter()
+{
+    echo "----- Installing troubleshooter -----"
+
+    # create temp directory
+    mkdir -p $TST_EXTRACT_DIR
+    cd $TST_EXTRACT_DIR
+
+    # grab tst bundle
+    wget $TST_PKG > /dev/null 2>&1
```

Comment:
Is "wget" a backup option in case we could not install from the setup files or is it a main installation option?
I remember it was discussed briefly during the initial review, but I was not sure which mode is the main one.
Overall, starting with network may not be a good option - in a fairly significant number of cases certain URLs would be blocked by customer VM firewall and we won't be able to install t-shoot tool to find that firewall is an issue :)

This comment may apply to the places in t-shoot tool code that rely on certain URLs being available (such as docs/KB/TSGs download). We can move such code behind connection checks where possible.

Reply:
Also, the tool would, probably, need to disclose all external locations we use to download pieces we need. E.g. in the initial dialog it presents or before certain t-shoot option is executed. You can follow up with Luke on this one to see if this is still a requirement. It used to be couple years ago when we delivered t-shoot tool for Azure Monitor's Status Monitor.

Reply:
That's valid, I originally had it that the 'wget' was the main installation (in case the installation of the troubleshooter failed), but it might actually be better if I have the main way of installing it be normally through the files, and only try to use the wget link OMS installation fails in some way

Reply:
I updated it so `wget` is used as a backup only if the troubleshooter isn't on the machine (see https://github.com/microsoft/OMS-Agent-for-Linux/pull/1190#issuecomment-663785862 for more detail), and if the `wget` doesn't work I have it print out the link needed to manually install the troubleshooter (aka the same link that didn't work), which should cover installation documentation

Reply:
I believe that inside the troubleshooter, all of the external links that I reference for information (i.e. not for connection checks) are all from that one Github repo (and the error message prints out the link that couldn't be reached on failure), but do you think I should add some more information at the start of the troubleshooter or something along those lines?